docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -66,7 +66,13 @@ Install Einstein for Developers:
 
 - If you are using Code Builder, click the Extensions icon in the Activity Bar in Code Builder, search for “Einstein for Developers” and click **Install**.
 
-**Note**: To use Einstein for Developers locally, you must have the [Salesforce Extensions pack]() installed in your VS Code for desktop application. See [Install Salesforce Extensions](https://developer.salesforce.com/tools/vscode/en/vscode-desktop/install) for more information.
+- You may run into some odd generated code outcomes if you have multiple AI-enabled extensions installed in VS Code. We recommend you disable all other AI-enabled extensions when working with Einstein for Developers:
```

Comment:
What about "unexpected" generated code outcomes instead of "odd" - which has multiple meanings.